app/controllers/admin/lookup_advisers_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Admin::LookupAdvisersController < Admin::ApplicationController
```

Comment:
Can we namespace these `LookupXXXController` under `Lookup` and take it out of the controller names please?